### PR TITLE
Add missing examples in Layout cops documentation

### PR DESCRIPTION
**Pull request by haziqhafizuddin**
Few missing examples in cop documentation. This commit adds the some missing example `layout` cops.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 * [#4444](https://github.com/bbatsov/rubocop/pull/4444): Make `Style/Encoding` cop enabled by default. ([@deivid-rodriguez][])
 * [#4452](https://github.com/bbatsov/rubocop/pull/4452): Add option to `Rails/Delegate` for enforcing the prefixed method name case. ([@klesse413][])
+* Add missing examples in `Layout` cops documentation. ([@haziqhafizuddin][])
```
Comment (Drenmi) [inline CHANGELOG.md]: We generally only add changelog entries for changes that affect behaviour for the end user. 🙂

File: lib/rubocop/cop/layout/else_alignment.rb [hunk 2]
```diff
@@ -7,6 +7,29 @@ module Layout
       # be aligned with an if/unless/while/until/begin/def keyword, but there
       # are special cases when they should follow the same rules as the
       # alignment of end.
+      #
+      # @example
+      #   # bad
+      #     # case 1
```
Comment (Drenmi) [inline lib/rubocop/cop/layout/else_alignment.rb]: I think no need to add additional nesting here, just split into separate `# bad` sections. 🙂

File: lib/rubocop/cop/layout/space_before_block_braces.rb [hunk 3]
```diff
@@ -5,6 +5,16 @@ module Cop
     module Layout
       # Checks that block braces have or don't have a space before the opening
       # brace depending on configuration.
+      # @example
```
Comment (Drenmi) [inline lib/rubocop/cop/layout/space_before_block_braces.rb]: +1 newline before `@example`. 😊

File: lib/rubocop/cop/layout/space_before_comment.rb [hunk 4]
```diff
@@ -5,6 +5,12 @@ module Cop
     module Layout
       # This cop checks for missing space between a token and a comment on the
       # same line.
+      # @example
```
Comment (Drenmi) [inline lib/rubocop/cop/layout/space_before_comment.rb]: +1 newline before example. 😊

File: lib/rubocop/cop/layout/space_before_semicolon.rb [hunk 5]
```diff
@@ -4,6 +4,12 @@ module RuboCop
   module Cop
     module Layout
       # Checks for semicolon (;) preceded by space.
+      # @example
```
Comment (Drenmi) [inline lib/rubocop/cop/layout/space_before_semicolon.rb]: +1 newline before example. 😊

File: lib/rubocop/cop/layout/space_before_comment.rb [hunk 6]
```diff
@@ -5,6 +5,12 @@ module Cop
...
+      #   # bad
+      #     1 + 1# this operation does ...
```
Comment (Drenmi) [inline lib/rubocop/cop/layout/space_before_comment.rb]: Code should be indented to same level as `# bad` and `# good` here. 🙂

**Comment by Drenmi** [comment]
Great work! 😊

**Comment by haziqhafizuddin** [comment]
thank you @Drenmi changes applied.

**Comment by bbatsov** [comment]
Lovely! Keep those improvements coming! 👍

**Pull request merged**